packages/playground-examples/copy/pt/JavaScript/Functions with JavaScript/Generic Functions.ts

```diff
@@ -0,0 +1,100 @@
+// Tipos genéricos provêm uma forma de usar Tipos como variáveis em outros
+// tipos. Meta.
```

Comment:
This sentence feels unnatural. I believe it can be improved:

> Tipos genéricos permitem usar Tipos como variáveis em outros tipos. Meta.

Reply:
Totally agree